spec/requests/api/v1/bank_accounts/index_spec.rb

```diff
@@ -18,19 +18,23 @@
     it 'returns bank_accounts' do
       expect(response).to be_success
 
-      expect(json[0]).to include(
-        'id'       => bank_account2.id,
-        'name'     => bank_account2.name,
-        'currency' => bank_account2.currency,
-        'balance'  => bank_account2.balance.as_json,
-        'residue'  => bank_account2.residue.as_json,
+      expect(json).to include(
```

Comment:
@yunixon not sure why do you need this fix. `json` should have array of bank_accounts. Why you change to compare `json` instead `json[0]`?